charts/operator/templates/deployment.yaml

```diff
@@ -25,14 +25,20 @@ spec:
           image: {{ printf "%s:%s" .Values.image.repository (default .Chart.AppVersion .Values.image.tag) }}
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           env:
-            - name: CONSUL_HTTP_ADDR
```

Comment:
as above, to make upgrade easier, I am tempted to keep this here for at least one release, then remove it when we bump to v1.0?